tests/std/tests/GH_001956_stl_verify_emits_long_strings/test.compile.pass.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
IIUC this test will be pointless after the release fixing DevCom-1437715 ships, so I think it should be removed.